conf/log.conf.d/httpd.portal.conf.example

```diff
@@ -1,56 +1,31 @@
 ### httpd.portal logger ###
-log4perl.rootLogger = INFO, HTTPD_PORTAL
+#log4perl.rootLogger = INFO, PFSYSLOG
```

Comment:
Because it inherits from the root logger in log.conf.
This is just a place to override it if required.
But I can see your point.
I will give it a separate configuration from the start, even if it's the same as the root.